packages/engine/Source/Shaders/Voxels/VoxelFS.glsl

```diff
@@ -182,7 +185,7 @@ void main()
         // This is similar to traverseOctreeFromBeginning but is faster when the ray is in the same tile as the previous step.
         positionUvShapeSpace = convertUvToShapeUvSpace(positionUv);
         traverseOctreeFromExisting(positionUvShapeSpace, traversalData, sampleDatas);
-        step = getStepSize(sampleDatas[0], viewRayUv, shapeIntersection);
+        step = getStepSize(sampleDatas[0], viewRayUv, shapeIntersection, currT);
```

Comment:
I know it's internal, but we try to avoid abbreviations where possible. Can this become `currentT`?